lib/node_modules/@stdlib/fs/append-file/README.md

```diff
@@ -0,0 +1,255 @@
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Append File
+
+> Append data to a file.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var appendFile = require( '@stdlib/fs/append-file' );
+```
+
+#### appendFile( file, data\[, options], clbk )
+
+Asynchronously appends `data` to a `file`.
+
+```javascript
+var join = require( 'path' ).join;
+
+var fpath = join( __dirname, 'examples', 'fixtures', 'file.txt' );
+
+appendFile( fpath, 'beep boop\n', onAppend );
+
+function onAppend( error ) {
+    if ( error ) {
+        console.log( error instanceof Error );
+        // => false
+    }
+}
+```
+
+The `data` argument may be either a `string` or a [`Buffer`][@stdlib/buffer/ctor].
+
+```javascript
+var join = require( 'path' ).join;
+var string2buffer = require( '@stdlib/buffer/from-string' );
+
+var fpath = join( __dirname, 'examples', 'fixtures', 'file.txt' );
+
+appendFile( fpath, string2buffer( 'beep boop\n' ), onAppend );
+
+function onAppend( error ) {
+    if ( error ) {
+        console.log( error instanceof Error );
+        // => false
+    }
+}
+```
+
+The function accepts the same `options` and has the same defaults as [`fs.appendFile()`][node-fs].
+
+#### appendFile.sync( file, data\[, options] )
+
+Synchronously appends `data` to a `file`.
+
+```javascript
+var join = require( 'path' ).join;
+
+var fpath = join( __dirname, 'examples', 'fixtures', 'file.txt' );
+
+try {
+    appendFile.sync( fpath, 'beep boop\n' );
+} catch ( error ) {
+    console.log( error.message );
+}
+```
+
+The function accepts the same `options` and has the same defaults as [`fs.appendFileSync()`][node-fs].
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+<!-- run-disable -->
+
+```javascript
+var fs = require( 'fs' );
+
+/* eslint-disable node/no-sync */
+
+// Check for directory path existence to prevent an error being thrown...
+if ( fs.existsSync( '/path/to' ) ) {
+    fs.appendFileSync( '/path/to/file.txt', 'beep boop\n' );
+}
+
+/* eslint-enable */
+
+```
+
+can be replaced by an approach which addresses existence via `error` handling.
+
+<!-- run-disable -->
+
+```javascript
+var appendFile = require( '@stdlib/fs/append-file' );
+
+// Explicitly handle the error...
+try {
+    appendFile.sync( '/path/to/file.txt', 'beep boop\n' );
+} catch ( error ) {
+    console.log( error.message );
+}
+```
+
+</section>
+
+<!-- /.notes -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var join = require( 'path' ).join;
+var appendFile = require( '@stdlib/fs/append-file' );
+
+var fpath = join( __dirname, 'examples', 'fixtures', 'file.txt' );
+
+// Synchronously append data to a file:
+try {
```

Comment:
Why are we wrapping in a try/catch when we know the function won't throw errors?

Reply:
fixed, now in line with the new sync implementation